### DISCUSSION TITLE
Heimdall LXC - HTTPS access behind reverse proxy

### DISCUSSION THREAD

**Discussion by BlackrazorNZ** [tteck/Proxmox#1518] [Help Needed] [answered]
Hi there. Love the scripts, hope you enjoy the coffee :) 

I’ve moved most of my self hosted items behind an NGINX proxy, and the only one of the LXC instances that has me stumped is Heimdall. I know 7990 is the default HTTP port but I can’t find (or see if is configurable to monitor) an HTTPS port. 

Any advice where to look to configure this? All the searches I’ve found talk about Heimdall in Docker instances and the methods aren’t relevant for the LXC. 

Thanks!

**Comment by tteck** [comment, marked as answer]
https://github.com/linuxserver/Heimdall/discussions/923

**Reply by BlackrazorNZ** [reply]
Thanks, that appears to be instructions for getting it working when both Heimdall and NGINX are on the same LXC, I have them on two different LXC’s with different IP’s but I’ll see if I can reverse engineer some of the concepts and apply them. Appreciated!
